Update/some_004.txt

```diff
@@ -3164,13 +3164,13 @@ void main()
 //r誰もいなくなったみたいに、ガランとした空間。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "誰もいなくなったみたいに、ガランとした空間。",
-			NULL, "It became an empty space, like everyone was gone.", Line_Normal);
+			NULL, "It became an empty space, like everyone else had disappeared.", Line_Normal);
 	ClearMessage();
 
 //r私の目に映るのは、……ただ、藤堂くん一人だけ…。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "私の目に映るのは、……ただ、藤堂くん一人だけ…。",
-			NULL, "In my eyes, all I could see...... Toudou-kun was the only thing here...", Line_Normal);
+			NULL, "In my eyes, all I could see...... was nobody else but Toudou-kun...", Line_Normal);
```

Comment:
```suggestion
			NULL, "In my eyes, all I could see...... was Toudou-kun. No one else...", Line_Normal);
```
Seeing a negative — "all I could see was nobody" — can make for an awkward sentence. Suggestion tries to clear that up.